core-services/image-mirroring/apiserver-network-proxy/mapping_apiserver-network-proxy_4_11

```diff
@@ -0,0 +1 @@
+registry.ci.openshift.org/ocp/4.11:apiserver-network-proxy quay.io/hypershift/apiserver-network-proxy:4.11 quay.io/hypershift/apiserver-network-proxy:latest
```

Comment:
given you're also putting it under the openshift org, why is it needed here?

Reply:
`kube-apiserver-network-proxy` would give better alignment to existing names, i think, unless this is generically useful for any apiserver.

Reply:
I think this is autogenerated. Renaming it sgtm, although it isn't really specific to the kube-apiserver but used by anything that needs to access the pod or service network. The current name is simply the name of the upstream project: https://github.com/kubernetes-sigs/apiserver-network-proxy

Reply:
autogenerated from what?

i'm still not sure why we'd mirror the same image twice, to two different repo orgs.

Reply:
IMO: mirroring into hypershift namespace may not needed after we ship into the release payload I feel.

@Basavaraju-G can you please remove the entry from the hypershift mirror and rerun the make to see if it still adds an entry, if yes - we need to find what is that config which is adding this automated entry.

Reply:
that worked manju. it added to only ocp mapping not hypershift one.